browser/src/AboutPage.tsx

```diff
@@ -2,18 +2,31 @@ import React from 'react'
 import styled from 'styled-components'
 import { PageHeading } from '@gnomad/ui'
 
+// @ts-expect-error TS(2307) FIXME: Cannot find module '../about/about.md' or its corr... Remove this comment to see the full error message
```

Comment:
Wonder why this couldn't be found.

Reply:
I think it's to do with the fact that this is a Markdown document rather than TS source, and `tsc` doesn't seem to understand the idea of including assets--at least not as currently configured. If you grep for `Cannot find module`, you'll see references to different sorts of assets: graphics, JSON, fonts, etc., as well as Markdown.

There should be a way to change the configuration to get rid of these warnings, but I left it for now since the build overall seems to work despite the warnings.